opennms-dao/pom.xml

```diff
@@ -114,7 +114,6 @@
     <dependency>
       <groupId>org.opennms.features.collection</groupId>
       <artifactId>org.opennms.features.collection.persistence.rrd</artifactId>
-      <scope>test</scope>
```

Comment:
Maybe we could move that method to the ResourcePath class itself instead? i.e. `resourcePath.toToFilesystemPath()`